docs/Recipes.md

```diff
@@ -0,0 +1,44 @@
+# Recipes
+
+## Appending Kakoune's shell prompt to the usual prompt
+
+If you are using `bash`, you can add the following lines to your `.bashrc`.
+
+``` sh
+if [ -e ~/.local/share/kak/connect/prompt ]; then
+    PS1="\$(~/.local/share/kak/connect/prompt)$PS1"
+fi
+```
+
+## [`:batch`][:batch] Integration to rename and drop buffers from the buffer list
+
+`~/.config/kak/connect/commands/:batch`
```

Comment:
There is a new option for plugins to register their paths.  It can also be used by users to configure custom paths.

Reply:
I'm going to remove it now that it is listed under integrations and the link to the code is in the readme.